Destination/Destination.php

```diff
@@ -3,15 +3,25 @@
 namespace Zenstruck\BackupBundle\Destination;
 
 use Psr\Log\LoggerInterface;
+use \Countable;
+use \IteratorAggregate;
 
 /**
  * @author Kevin Bond <kevinbond@gmail.com>
  */
-interface Destination
+interface Destination extends Countable, IteratorAggregate
```

Comment:
Yes, it is possible to do that with S3Cmd but we would have to parse the output of the command which would get ugly.

For my use case, I want it to be a black hole. I have policies on S3 that manage the backups for me (moving old ones to Glacier and deleting the really old ones). For me, the backup is fire and forget.

There is no risk/danger but for some destinations (like S3Cmd) it doesn't make sense.